### DISCUSSION TITLE
Use custom login.html in auth0 simulator

### DISCUSSION THREAD

**Discussion by rromanchenko** [thefrontside/simulacrum#189] [Q&A]
Hi Team,
Just wonder, if there is a way to use Auth0 simulator with whatever custom 'login.html' layout?
As far I can see, for now simulator uses some default 'login.html': https://github.com/thefrontside/simulacrum/blob/v0/packages/auth0/src/views/login.ts#L20

Thanks.

**Comment by cowboyd** [comment]
@rromanchenko This would be a really great feature to have. At the very least, perhaps we should make this a very generic "Simulated Auth0 Login Page" so that it isn't branded with our Logo.

I think the next step would be to figure out how to make this page pluggable (which it is currently not)
